pages/settings/privacy/index.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import Glean from '@mozilla/glean/web'
+import { mastodonAccountHandle, mastodonAccountId } from '~/telemetry/generated/identifiers'
+import { userAgent } from '~~/telemetry/generated/identifiers'
+
+const user = currentUser.value
+
+const { t } = useI18n()
+
+useHydratedHead({
+  title: () => `${t('settings.privacy.label')} | ${t('nav.settings')}`,
+})
+const userSettings = useUserSettings()
+
+function toggleOptOut() {
+  const allowGlean = togglePreferences('allowGlean')
+  Glean.setUploadEnabled(allowGlean)
+
+  // reset identifiers if user opts back in
+  if (user && allowGlean) {
+    mastodonAccountHandle.set(user.account.acct)
+    mastodonAccountId.set(user.account.id)
+    userAgent.set(navigator.userAgent)
+  }
+}
+</script>
+
+<template>
+  <MainContent back-on-small-screen no-beta-label>
+    <template #title>
+      <div text-lg font-bold flex items-center gap-2 @click="$scrollToTop">
+        <span>{{ $t('settings.interface.label') }}</span>
```

Comment:
This translation string should `settings.privacy.label` (Right now it renders as `Interface`)

Reply:
Good catch! fixed